generators/client/templates/react/src/main/webapp/app/app.tsx.ejs

```diff
@@ -50,7 +52,7 @@ export class App extends React.Component<IAppProps> {
   render() {
     const paddingTop = '60px';
     return (
-      <Router>
+      <Router basename={baseHref}>
```

Comment:
Required when deploying under a context-path other than `/`, fixes all links to include the base href ([docs](https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/docs/api/BrowserRouter.md#basename-string))